webapp/shop/api/ua_contracts/api.py

```diff
@@ -319,6 +319,15 @@ def post_purchase_calculate(
             error_rules=["default"],
         ).json()
 
+    def post_magic_attach(self, request_body: dict) -> dict:
+        self._request(
+            method="post",
+            path=f"v1/magic-attach/activate",
+            json=request_body,
+            error_rules=["default"],
+        )
+        return {"success": "true"}
```

Comment:
Should we not be looking at the API response to make sure the attach was indeed succesful?

Reply:
If the response from the API is empty, it is successful. So that value will be returned only if the attachment is successful